modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,30 @@
 [id="cnf-cpu-infra-container_{context}"]
 = Restricting CPUs for infra and application containers
 
-You can reserve cores (threads) from a single NUMA node for operating system housekeeping tasks and put your workloads on another NUMA node. Partitioning the CPUs this way can prevent the housekeeping processes from impacting latency-sensitive application processes. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
+Housekeeping and workloads may utilize the central processing units (CPUs) in a way that would impact latency sensitive processes running on those CPUs. Partitioning your workloads on separate CPUs prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
```

Comment:
I am not sure this is accurate now. What the admin does is really partitioning of the cpus and then assigning the processes (sensitive workload, vs. generic processes, workloads, interrupts and the operating system) to those partitions.

Reply:
It might be easier to understand from slides 8 and 9 here https://static.sched.com/hosted_files/devconfcz2021/70/Openshift-for-low-latency-and-real-time-workloads.pdf

Reply:
Thanks! I'll make the suggested changes. I'm also making slide 9 into a table, which might be easier for users to follow.